File: hello.go
package main

import "fmt"

type Greeter struct{}

func (g Greeter) Greet() {
    fmt.Println("Hello World")
}

func main() {
    greeter := Greeter{}
    greeter.Greet()
}
// Updated at Fri Sep 25 04:05:10 UTC 2026

File: hello.js
class Greeter {
    constructor(name) {
        this.name = name;
    }

    greet() {
        console.log(`Hello, World from JavaScript OOP! (Greetings to ${this.name})`);
    }
}

const greeter = new Greeter("GitHub Automation");
greeter.greet();
